lunatrace/bsl/hasura/metadata/databases/lunatrace/tables/public_organization_user.yaml

```diff
@@ -27,16 +27,14 @@ insert_permissions:
           - organization:
               creator_id:
                 _eq: X-Hasura-Real-User-Id
-          - _exists:
```

Comment:
The reason this exists like this is to allow the admin dashboard to make modifications to an organization. Reverting this will break that feature.